content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -94,10 +101,19 @@ In order to use it, cgroup v2 must be supported by the CRI runtime as well.
 Follow this [Migration guide](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/)
 if you wish to migrate to the `systemd` cgroup driver in existing kubeadm managed clusters.
 
+## CRI version support {#cri-versions}
+
+Your container runtime must support at least v1alpha2 of the container runtime interface.
```

Comment:
v1alpha2 is considered deprecated, should we add that v1alpha2 is deprecated or just recommend v1
reference: https://kubernetes.io/docs/concepts/architecture/cri/

>For Kubernetes v1.23, the kubelet prefers to use CRI v1. If a container runtime does not support v1 of the CRI, then the kubelet tries to negotiate any older supported version. The v1.23 kubelet can also negotiate CRI v1alpha2, but this version is considered as deprecated.

Reply:
How's this revised text?

Reply:
lgtm too